src/YOUR_NAME_ex/ex05/myrand.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Isak Finnøy'
+__email__ = 'isfi@nmbu.no'
+
+
+class LCGRand:
+
+    def __init__(self, seed):
+        """
+        :param seed: arbitrary input
+        """
+        self.seed = seed
+        self.a = 7 ** 5
+        self.m = 2 ** 31 - 1
```

Comment:
These should be class variables.
Please check the skeleton from the task requirement.